requirements/base.txt

```diff
@@ -1,11 +1,2 @@
-# Kolibri runtime requirements
-docopt==0.6.2
-colorlog==2.6.0
-django==1.9.7
-# we need some unreleased django-mptt fixes that came in after 0.8.4; can switch to 0.8.5 when released
-git+https://github.com/django-mptt/django-mptt.git@9e131f91a9992d0495e1217bdcf7036a5e11d8c5#egg=django-mptt
-djangorestframework==3.3.3
-six==1.10.0
-django-js-reverse==0.7.2
-django-filter==0.13.0
-cherrypy==6.0.2
+# Base requirements are empty. All of our minimum requirements are specified in setup.py's install_requires variable!
+-e .
```

Comment:
Have you checked what happens if you do `pip install` from a different working directory? E.g. from an absolute path? Will `.` still suffice, or is there a way of doing more robust relative paths?